app/ui/browser/reducers/main-reducers.js

```diff
@@ -11,11 +11,12 @@ specific language governing permissions and limitations under the License.
 */
 
 import assert from 'assert';
+import 'babel-polyfill';
```

Comment:
Chatted with Jordan and let's just land this for now and we can see if that becomes an issue